app/components/track-info/template.hbs

```diff
@@ -1,37 +1,12 @@
 <div class="dot"></div>
-{{#if (or ensembleName conductorName)}}
-  <NyprAToggleBox class="secondary-metadata-toggle-box" @horizontalPosition="right" as |toggle|>
-    <toggle.trigger>
-      <div class="metadata">
-        {{#if composerName}}
-          <div class="composer">{{composerName}}</div>
-        {{/if}}
-        {{#if trackTitle}}
-          <div class="title">{{trackTitle}}</div>
-        {{/if}}
-      </div>
-    </toggle.trigger>
-    <toggle.dropdown class="secondary-metadata-content-dropdown">
-      <div class="secondary-metadata-content">
-        {{#if ensembleName}}
-          <div class="ensemble secondary-metadata-content-field">Performed by the {{ensembleName}}</div>
-        {{/if}}
-        {{#if conductorName}}
-          <div class="conductor secondary-metadata-content-field">Conducted by {{conductorName}}</div>
-        {{/if}}
-      </div>
-    </toggle.dropdown>
-  </NyprAToggleBox>
-{{else}}
-  <div class="metadata">
-    {{#if composerName}}
-      <div class="composer">{{composerName}}</div>
-    {{/if}}
-    {{#if trackTitle}}
-      <div class="title">{{trackTitle}}</div>
-    {{/if}}
+
+<div class="metadata">
+  <div class="composer">{{composerName}}</div>
+  <div class="title">
+    <TrackAdditionalInfo @title={{trackTitle}} @ensembleName={{ensembleName}} @conductorName={{conductorName}} />
   </div>
-{{/if}}
+</div>
+
```

Comment:
One catch here -- Laura wanted clicks on both the composer and title to trigger the metadata popup.

Reply:
Also, and this could go into a separate ticket if need be, but the caret is supposed to line up with the end of the the title text (even when it wraps...). You can see it in the design here: https://app.zeplin.io/project/5d83c520c2bfab889196fdf1/screen/5d7f9b22fddef1182ddb41c4

Reply:
Ah dang, I missed the clicks on both of them. Figured it was just the last one since that's where it lined up. That might require some reworking… I'm gonna merge this for now and we can address that later.